code/datums/setup_option/core_implants.dm

```diff
@@ -34,6 +34,7 @@
 	restricted_jobs = list(
 		/datum/job/doctor,
 		/datum/job/recovery_team,
+		/datum/job/medstudent,
```

Comment:
You'll need to add the other learner roles to this list at well, since we don't want Blackshield Cadets with the cultured organ

Reply:
restricted_depts = SECURITY | BLACKSHIELD | PROSPECTORS | ENGINEERING | SCIENCE | CHURCH | INDEPENDENT | CIVILIAN | LSS | LODGE

Handled via this